index.js

```diff
@@ -109,6 +110,13 @@ app.event('message', async (args) => {
       .catch((e) => {
         console.warn(e)
       })
+    mirrorMessage({
+      message: `<@${user}> has been dropped into cave`,
+      user,
+      channel,
+      type,
+    })
+    scheduleHelpMeMessage(client, user)
```

Comment:
Seems like we have a couple async jobs here... how about we await them together?

```js
   const jobs = []
   jobs.push(client.chat.delete({
        token: process.env.SLACK_LEGACY_TOKEN, // sudo
        channel,
        ts,
      })
      .catch((e) => {
        console.warn(e)
      }))
    jobs.push(mirrorMessage({
      message: `<@${user}> has been dropped into cave`,
      user,
      channel,
      type,
    }))
    jobs.push(scheduleHelpMeMessage(client, user))
    
    await Promise.all(jobs)
```